source/reference/v2/customers-api/list-customers.rst

```diff
@@ -139,6 +139,16 @@ Example
       // Next page
       $customers->next();
 
+   .. code-block:: python
+      :linenos:
+
+      from mollie.api.client import Client
+
+      mollie_client = Client()
+      mollie_client.set_api_key('test_dHar4XY7LxsDOtmnkVtjNVWXLSlXsM')
+
+      customers = mollie_client.customers.list()
```

Comment:
Also add an example for getting the next page of paginated results (`get_next()`)